metron-interface/metron-alerts/src/app/alerts/meta-alerts/meta-alerts.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+
+import {routing} from './meta-alerts.routing';
+import { MetaAlertsComponent } from './meta-alerts.component';
+import {UpdateService} from '../../service/update.service';
+import {MetaAlertService} from '../../service/meta-alert.service';
+import {SharedModule} from '../../shared/shared.module';
+
+@NgModule({
+  imports: [ routing,  SharedModule ],
+  declarations: [ MetaAlertsComponent ],
+  providers: [ UpdateService, MetaAlertService ],
```

Comment:
Should UpdateService be in the list of providers here?  Isn't it application-wide and provided at the app root level as well?